Datahub.Core/Model/Datahub/PublicDataSharingRecords.cs

```diff
@@ -134,6 +134,7 @@ public class OpenDataSharedFile: SharedDataFile
         public FileStorageType? FileStorage_CD { get; set; }
         public OpenDataUploadStatus UploadStatus_CD { get; set; }
         public string UploadError_TXT { get; set; }
+        public string FileUrl_TXT { get; set; }
```

Comment:
I could make it nullable but that will create a Warning because this whole project has not enabled nullables. If I make the file _#nullable enable_ then we'll get a ton Warnings more. With strings this is safe, since they are objects and nullable for the database purpose. I agree any new project we should enable nullable globally and avoid any warning.

Reply:
Makes sense, we should look at enabling nullables for all projects but not urgent. Lets figure that out later.